scripts/global.js

```diff
@@ -1002,22 +1002,35 @@ async function renderProposals(arrProposals, fContested) {
 
         // Proposal Status calculation
         const nRequiredVotes = cMasternodes.enabled / 10;
+        const nMonthlyPayment = parseInt(cProposal.MonthlyPayment);
         let strStatus = '';
         let strFundingStatus = '';
+        let strColourClass = '';
```

Comment:
Initialize 
`let strColourClass = 'No';`
instead of copy and pasting it in every if branch

Reply:
Done